jest.config.ts

```diff
@@ -5,11 +5,6 @@ const jestConfig: JestConfigWithTsJest = {
   moduleNameMapper: {
     "^(\\.{1,2}/.*)\\.js$": "$1",
   },
-  testMatch: ["**/test/**/*.test.ts", "**/test/tests.ts"],
-  coverageProvider: "v8",
```

Comment:
Can you elaborate on why you want this to be changed and why we should? We set this to V8, but this would use babel again.

Reply:
I checked the documentation and some blog posts, but I could not find any distinct advantages deviating from the default here. If there is a good reason to keep it I see no problem restoring this, WDYT?

Reply:
v8 would use native coverage in node, which I tend to prefer and use in most of the libraries I maintain. I don't feel super strong about this, but I also feel like we shouldn't just change things only for the sake of changing them.